app/controllers/events_controller.rb

```diff
@@ -70,4 +76,8 @@ def event_search_params
     (params[Events::Search.model_name.param_key] || defaults)
       .permit(:type, :distance, :postcode, :month)
   end
+
+  def cap_results?
+    params.dig("events_search").blank?
```

Comment:
Does this return true or false when someone does an 'empty search'? i.e. if they search but using the default selections of all types and nationwide? I _think_ we would want to cap in this scenario, but may be worth double checking with Myles. Either way it would be good for a test to clarify I think?

Reply:
Just seen this - hadn't really seen this controller code before - would it make sense to either move it to the Search model, or better create a Categories PORO model - that could be initialised from an API response. And then the logic of the categorisation can get tested at unit test level?

Reply:
@jebw - think that might be phase two, I'm definitely planning on revisiting this controller with regards to splitting out some of the more complex functionality and generally thinning it out a bit. This seemed like a high priority fix to prevent the events page from being exceedingly long.

Reply:
@ethax-ross - yeah, Myles agreed. Just pushed with the `#cap_results?` logic updated. Thanks